tests/modules/extra/slack/integration/test_slack_notifier.py

```diff
@@ -0,0 +1,41 @@
+import os
+from unittest import skipIf
+
+import pytest
+
+from petisco import NotifierExceptionMessage, NotifierMessage, UnknownError
+from petisco.extra.slack import SlackNotifier
+
+LONG_TEXT = "long_text" * 1000
+
+
+@skipIf(
```

Comment:
I guess this works perfectly, however I'd use `@pytest.mark.skipif` to be consistent with another tests (e.g https://github.com/alice-biometrics/petisco/blob/e8b468ee74b6839fa6b1ff85307c619230030c39/tests/modules/extra/testing_decorators.py#L37)

Reply:
Done!